tagmanager/src/tm_tag.h

```diff
@@ -80,7 +80,10 @@ typedef enum
 #define TAG_IMPL_VIRTUAL 'v' /**< Virtual implementation */
 #define TAG_IMPL_UNKNOWN 'x' /**< Unknown implementation */
 
-typedef struct _TMTag
+/**
+ * The TMTag structure represents a single tag in the tag manager.
+ **/
+typedef struct TMTag
```

Comment:
If it's not, please drop this.  yeah it's more consistent so that's kinda good, but it changes a name so it could break some users, and I'd rather not play with that so close to the release.
Yes I know it's private and all.